Clarify the semantics of reverse range lookup requests

kkaneda opened the pull request:
A reverse lookup request returns a range containing the previous key of the requested key (e.g., if a requested key is the end key of range R, the reverse lookup request returns R).

tschottdorf commented on proto/api.proto:
```diff
@@ -338,12 +338,16 @@ message AdminMergeResponse {
+// RangeLookupRequest also specifies the maximum number of range
+// descriptors that should be returned, if there are additional
+// consecutive addressable ranges. Specify max_ranges > 1 to pre-fill the
+// range descriptor cache.
```
you could add that the additional ranges "go in the direction of" the scan (not sure how to word it best).

kkaneda replied:
Sure, added the following sentence:

> The additional ranges are scanned in the same direction as lookup (forward v.s. reverse).

tschottdorf commented:
LGTM

mrtracy commented on proto/api.proto:
```diff
@@ -338,12 +338,16 @@ message AdminMergeResponse {
-// ranges. Specify max_ranges > 1 to pre-fill the range descriptor
-// cache.
+// A RangeLookupRequest is arguments to the RangeLookup() method. A
+// non-reverse lookup request returns a range containing the requested
```
s/non-reverse/forward/ ?

kkaneda replied:
Fixed!

mrtracy commented:
LGTM

kkaneda committed:
Clarify the semantics of reverse range lookup requests

A reverse lookup request returns a range containing the previous key
of the requested key (e.g., if a requested key is the end key of range
R, the reverse lookup request returns R).
